### ISSUE TITLE
curator app: when adding tree to a collection we should show the tree name not the ID to the user

### ISSUE THREAD

**Issue by mtholder** [OpenTreeOfLife/opentree#1026]

**Comment by mtholder** [comment]
oops. it looks like the name of the tree is getting "frozen" in the collection such that updating the name in the study does not update the collection.

**Comment by jimallman** [comment]
If I recall correctly, we don't get tree labels (names) from oti, just ids. This should be easy to fix once we've replaced this with the new ottreeindex.

Hm, I do see `ot:originalLabel` listed among the [response properties for `/v3/studies/find_trees`](https://github.com/OpenTreeOfLife/germinator/wiki/Studies-API-v3#find_trees), and it is among the listed `tree_properties` returned from `/v3/studies/properties`:
```bash
curl -X POST https://devapi.opentreeoflife.org/v3/studies/properties
```
...but this property never seems to show up in use. (As a test, I named a couple of trees in [Martin, 2014](https://devtree.opentreeoflife.org/curator/study/edit/pg_2606?tab=trees) on **devtree**, and still this property does not appear.)

**Comment by jimallman** [comment]
> it looks like the name of the tree is getting "frozen" in the collection such that updating the name in the study does not update the collection.

Yes, this is a known issue: We store study/tree properties at the time a tree is added, and we don't update these values to match ongoing changes in the study. There's an issue for this, and some discussion about whether to cache+update this, or always fetch the latest labels and status when displaying a tree collection.

**Comment by jimallman** [comment]
I believe this was addressed in a recent pull request #1219. Please confirm (and close this) or add any changes that are still needed.
